pydomcfg/tests/bathymetry.py

```diff
@@ -44,6 +45,52 @@ def flat(self, depth: float) -> Dataset:
         ds["Bathymetry"] = xr.full_like(ds["glamt"], depth)
         return _add_attributes(_add_mask(ds))
 
+    def sea_mount(self, depth: float, stiff: float = 1) -> Dataset:
+        """
+        Channel with seamount case.
+
+        Produces bathymetry of a channel with a Gaussian seamount in order to
+        simulate an idealised test case. Based on Marsaleix et al., 2009
+        doi:10.1016/j.ocemod.2009.06.011 Eq. 15.
+
+        Parameters
+        ----------
+        depth: float
+            Bottom depth (units: m).
+        stiff: float
+            Scale factor for steepness of seamount (units: None)
+
+        Returns
+        -------
+        Dataset
+        """
+        ds = self._coords
+
+        # Find half way point for sea mount location
+        half_way = {k: v // 2 for k, v in ds.sizes.items()}
+        glamt_mid, gphit_mid = (g.isel(half_way) for g in (ds.glamt, ds.gphit))
+
+        # Define sea mount bathymetry
+        ds["Bathymetry"] = depth * (
+            1.0
+            - 0.9
+            * np.exp(
+                -(
+                    stiff
+                    / 40.0e3 ** 2
+                    * ((ds.glamt - glamt_mid) ** 2 + (ds.gphit - gphit_mid) ** 2)
+                )
+            )
+        )
+
+        # Add rmax of Bathymetry
+        # ds["rmax"] = DataArray(
+        #    _calc_rmax(ds["Bathymetry"].to_masked_array()), dims=["y", "x"]
+        # )
+        ds["rmax"] = _calc_rmax(ds["Bathymetry"])
+
+        return _add_attributes(_add_mask(ds))
```

Comment:
Sorry, I did something pretty bad here.
This was actually introduced by @jdha.
I merged main and committed locally in a single commit, so GH is not realizing that it's the same code (I just added the type annotation).